src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -247,6 +248,35 @@
               + "cachable actions that output symlinks will fail.")
   public boolean allowSymlinkUpload;
 
+  /**
+   * Describes what kind of remote build outputs to download locally.
+   */
+  public enum RemoteOutputsStrategy {
+    /** Download all remote outputs locally. */
+    ALL,
+    /** Generally don't download remote outputs. */
+    MINIMAL,
+  }
+
+  public static class RemoteOutputsStrategyConverter extends EnumConverter<RemoteOutputsStrategy> {
+    public RemoteOutputsStrategyConverter() {
+      super(RemoteOutputsStrategy.class, "download remote outputs");
+    }
+  }
+
+  @Option(
+      name = "experimental_remote_outputs",
```

Comment:
just a suggestion: expertimental_remote_output_strategy
From my foreign perspective I cannot see what a flag `experimental_remote_outputs` is responsible for without reading a documentation.

Reply:
How about `--remote_download_outputs`?

Reply:
I like this better than initial name :+1:

Reply:
done